packages/exp/src/expo_commands/build-webhook.js

```diff
@@ -0,0 +1,108 @@
+import { URL } from 'url';
+import { BuildWebhook, Exp } from 'xdl';
+import chalk from 'chalk';
+
+import log from '../log';
+
+export default program => {
+  program
+    .command('build-webhook:set [project-dir]')
+    .option('--url <webhook-url>', 'Webhook to be called after building the app.')
+    .option(
+      '--secret <webhook-secret>',
+      'Secret to be passed as a query param (?secret=<webhook-secret>) to the Webhook. It has to be at least 16 characters long.'
```

Comment:
What's the reason for having `secret` as a separate option, since people will be able to add any query params they want to the webhook URL directly anyway (including query params with a different name than the hardcoded `secret` too)?

Maybe drop `--secret` and just have URL?

Reply:
We encrypt that secret in www and store in the database as a ciphertext. Yes, we could require users to pass it as query param, but then we'd need to parse that url in www.

Reply:
If encryption is desired, why don't we encrypt the whole URL then? They might have other query params in the URL that are also secrets, if their API endpoint takes a param named something else than `secret`.

Reply:
Well, we don't pass user's query parameters to the webhook call (we only pass our own parameters). If users want us to call their webhooks, they should follow our docs. So i don't agree that we would need to even consider encrypting the whole URL. 

If you would like to, i can add more validation in exp of webhook url. We can check whether a user passes some query params and if so, we could print an error "Query parameters are not allowed" or sth similar.

Also, please check James's comment - https://github.com/expo/universe/issues/2979#issuecomment-415534770